include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -370,7 +370,7 @@ struct Value : Base<OrtValue> {
   const T* GetTensorData() const;
 
   template <typename T>
-  T At(const std::initializer_list<size_t>& location);
+  T At(const std::initializer_list<int64_t>& location);
```

Comment:
>std::initializer_list [](start = 13, length = 21)

Should be make it a std::vector by value so it be moved inside. This way it would be more flexible.